mcs_benchmark_data/loaders/rdf_file_loader.py

```diff
@@ -1,17 +1,18 @@
 from pathlib import Path
-from typing import Optional
+from typing import Optional, Generator
 
 from pathvalidate import sanitize_filename
 from rdflib import Graph
 import bz2
 
 from tqdm import tqdm
 
-from mcs_benchmark_data.loaders._buffering_loader import _BufferingLoader
+from mcs_benchmark_data._loader import _Loader
 from mcs_benchmark_data.namespace import bind_namespaces
+from mcs_benchmark_data._model import _Model
 
 
-class RdfFileLoader(_BufferingLoader):
+class RdfFileLoader(_Loader):
```

Comment:
The RdfFileLoader doesn't actually need to be a buffering loader in this project. In paradicms it does for reasons that aren't relevant here.

It can be:

```
def load(self, *, models: Generator[_Model, None, None):
    ... instantiate and bind graph  as currently...
    for model in models:
       model.to_rdf(graph=graph)
    ... write graph to disk as currently ...
```

You don't need flush, you don't need to keep the __models buffer.